tests/blackbox_test.go

```diff
@@ -38,6 +40,16 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+type Luks struct {
+	Name string `json:"name"`
+}
+
+type IgnitionConfig struct {
+	Storage struct {
+		Luks []Luks `json:"luks,omitempty"`
+	} `json:"storage"`
+}
```

Comment:
Hmm, so I opted not to because I only needed 1 field and our types are versioned, i.e config 3.4 or so which is awkward because these tests deal with old and new versions. By creating this non version'd type and scraping what I need I explicitly indicate the data I care about. 

it was just a choice; I dont know if it was the right one lmk!

Reply:
In tandem with the other comment about moving this to `outer()`, I noticed that function already also parses the Ignition config, so we should be able to leverage that parsed object instead there in our deferred code block.